### DISCUSSION TITLE
What does a dot means in "Resolving dependencies...."?

### DISCUSSION THREAD

**Discussion by dorianmarie** [ruby/rubygems#7401] [Q&A] [answered]
Just curious, sometimes I have few dots, sometimes I have a lot of dots

Related: https://stackoverflow.com/questions/56890732/what-do-dots-after-resolving-dependencies-from-bundle-install-or-bundle-updat

**Comment by simi** [comment, marked as answer]
If I remember well, those additional dots are coming from fetchers and they do usually report network activity.

https://github.com/rubygems/rubygems/blob/c4b91b17d6025b9927702dfa2da9f27252dfeeed/bundler/lib/bundler/fetcher/base.rb#L47

You can turn each dot into actual log line by specifying `BUNDLER_DEBUG_RESOLVER=1` in your env.

**Reply by dorianmarie** [reply]
Thanks <3
